src/index.ts

```diff
@@ -115,6 +115,8 @@ export * from "./repository/TreeRepository"
 export * from "./repository/MongoRepository"
 export * from "./repository/RemoveOptions"
 export * from "./repository/SaveOptions"
+export { UpdateOptions as RepositoryUpdateOptions } from "./repository/UpdateOptions"
```

Comment:
There's a conflict with UpdateOptions from Mongo typings,
what do you think we should do about this? 🤔